deploy/chart/templates/0000_50_olm_00-namespace.yaml

```diff
@@ -2,9 +2,17 @@ apiVersion: v1
 kind: Namespace
 metadata:
   name: {{ .Values.namespace }}
+  labels: 
+    pod-security.kubernetes.io/enforce: restricted
```

Comment:
In Openshift the label syncer can change that. 
Therefore, that means the solution proposed here will fail when:

- The Dockerfile image does not define the USER ID
AND
- The Pod be created with RunAsNonRoot
AND
- The label syncer changes the enforcement of this namespace  

@perdasilva @anik120